app/javascript/controllers/tab_nav_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from '@hotwired/stimulus'
+import * as bootstrap from 'bootstrap'
+
+export default class extends Controller {
+  static values = { selector: String }
+
+  showNext (event) {
+    event.preventDefault()
+
+    const nextTab = this.currentTab.nextElementSibling
+    bootstrap.Tab.getOrCreateInstance(nextTab).show() // eslint-disable-line no-undef
+  }
+
+  showPrevious (event) {
+    event.preventDefault()
+
+    const previousTab = this.currentTab.previousElementSibling
+    bootstrap.Tab.getOrCreateInstance(previousTab).show() // eslint-disable-line no-undef
+  }
+
+  get currentTab () {
+    return document.querySelector(`#${this.selectorValue}`)
```

Comment:
Trivial, but: do you think adding the `#` here makes more sense than when setting the selector value in rubyland? This handles the concern in one place (vs. three), but it seems a little unexpected to set a selector value in rubyland that is inherently missing key information. It isn't really a fully useful selector when set in Rubyland.

Either way's fine, just curious.